data/strings/strings.txt

```diff
@@ -49364,3 +49364,21 @@
   [taxi_category_order]
     en = Order
     ru = Заказать
+
+  [arsenal_telemed_map_downloader_title]
+    en = Онлайн консультации профессиональных врачей 24/7
```

Comment:
note: we must have an "en" translation so this is a workaround, the correct locale is used in runtime